SpecRunner.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+
+  <!-- include source files here... -->
+  <script src="../../../week7/CLICTestFramework/testing-framework.js"></script>  
```

Comment:
I couldn't get the specrunner to work and I think it might be because of this path here 🤔